Dockerfile

```
@@ -1,33 +1,44 @@
+    && rm -rf /var/lib/apt/lists/*
+COPY prisma.config.ts ./prisma.config.ts
+COPY prisma ./prisma
+CMD pnpm exec prisma migrate deploy
+
+FROM base AS runner
+ENV NODE_ENV=production
+ENV PORT=3000
+ENV HOSTNAME=0.0.0.0
+ENV DATABASE_URL="file:/app/data/prod.db"
+
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)